### PR TITLE
[BitSail][Docs] reorganize community documents structure and add more guide documents

### PR DESCRIPTION
**Pull request by love-star**
Signed-off-by:

## Pre-Checklist

Note: Please complete **_ALL_** items in the following checklist.

- ✅ I have read through the [CONTRIBUTING.md](https://github.com/bytedance/bitsail/blob/master/docs/contributing.md) documentation.
- ✅ My code has the necessary comments and documentation (if needed).
- ✅ I have added relevant tests.

## Purpose

reorganize community documents structure and add more guide documents.

- add `BitSail Guide Video` under Documents.
- add `Pull Request guide` under Community.
- add `BitSail Release Guide` under Community.
- add `Connector Quick Start` under Community.
- add `Source Connector Details` under Community.
- add `Sink Connector Details` under Community.

N/A

### CODE REVIEW

File: website/en/documents/start/release_guide.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+[Common] bitsail-common
+[Core] base client component cores
+[Connector] all connector related changes
+[Doc] documentation or java doc changes
+[Build] build, dependency changes
+```
+
+## Procedure to release
+
+![img](website/images/documents/start/release_guide/release_procedure.png)
```
Comment (BlockLiu) [inline website/en/documents/start/release_guide.md:35]: Image was broken.

File: website/en/documents/start/release_guide.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+Once all issues have been resolved, you should go back and build a new release candidate with these changes.
+
+### 6. Finalize the release
+
+Once the release candidate passes the voting, we could finalize the release.
+
+A. Change the release branch version from x.x.x-rc1 to x.x.x. e.g. 0.1.0-rc1 to 0.1.0
+
+B. git commit -am "[MINOR] Update release version to reflect published version  ${RELEASE_VERSION}"
```
Comment (BlockLiu) [inline website/en/documents/start/release_guide.md:91]: You can quote such command with "``"

File: website/en/documents/start/source_connector_detail.md [hunk 3]
```diff
@@ -0,0 +1,1545 @@
+# Source Connector Details
```
Comment (BlockLiu) [inline website/en/documents/start/source_connector_detail.md:1]: Title is not compatible with sink connector.

File: website/zh/documents/start/README.md [hunk 4]
```diff
@@ -8,4 +8,9 @@ dir:
 
  - [开发环境配置](env_setup.md)
  - [部署指南](deployment.md)
- - [任务配置指南](config.md)
\ No newline at end of file
+ - [任务配置指南](config.md)
+ - [BitSail 实机演示](quick_guide.md)
```
Comment (BlockLiu) [inline website/zh/documents/start/README.md:12]: Same as english version.

**Comment by lichang-bd** [comment]
how about this task, no update for a long time @love-star

**Comment by love-star** [comment]
> how about this task, no update for a long time @love-star

I will continue this task after the metric gather task finish, what's more, I will add the document of metric gather at that time.

**Comment by love-star** [comment]
@BlockLiu I have solved all comments, please take a look when you are free.

**Comment by BlockLiu** [comment]
Nice PR.

**Pull request merged**